File: sandbox/python/src/TemplateListDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.TemplateApi(api_client).template_list(
            account_id=None,
            page=1,
            page_size=20,
            query=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Template#template_list: %s\n" % e)

File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateListDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateListDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        try
        {
            var response = new TemplateApi(config).TemplateList(
                accountId: null,
                page: 1,
                pageSize: 20,
                query: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Template#TemplateList: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
